svc-task/tmessagebus/publish.go

```diff
@@ -59,6 +59,5 @@ func Publish(taskURI string, messageID string, eventType string) {
 		log.Println("Unable Publish events to kafka", err)
 		return
 	}
-	log.Printf("debug: %s Event Published", messageID)
-
+	log.Println("info: TaskURI:", taskURI, ", EventID:", eventID, ", MessageID:", messageID, " of the published event")
```

Comment:
what is the log level log.Println will print here ? "info: " should not be there.

Reply:
remove "of the published event"
In the following what is **'I'** 
2020-11-10 08:01:05.810994 **I** | info: TaskURI: /redfish/v1/TaskService/Tasks/taske5838831-46f3-44da-be7f-8434e36c8b16 , EventID: 8f80c321-5781-4ac2-a1c5-7ee04ad0feb0 , MessageID: TaskEvent.1.0.1.TaskCompletedOK  of the published event

Reply:
remove **of the published event"** from the log

Reply:
updated